toplevel/usage.ml

```diff
@@ -94,6 +94,8 @@ let print_usage_common co command =
 \n                          for full Gc stats dump)\
 \n  -bytecode-compiler (yes|no)        enable the vm_compute reduction machine\
 \n  -native-compiler (yes|no|ondemand) enable the native_compute reduction machine\
+\n  -nI dir                OCaml include directories for the native compiler (default if not set) \
+\n  -native-output-dir     (none|<directory>) set the output directory for native objects\
```

Comment:
What is the difference between `none` and `.`?

Reply:
I guess there is no difference, we could try that indeed.